auslib/db.py

```diff
@@ -20,7 +20,7 @@
 import dictdiffer
 import dictdiffer.merge
 
-from auslib.global_state import cache, dbo
```

Comment:
Yeah, we can't get rid of it entirely - it's required by the web layer. Good thought though!